package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "circleci-docs",
+  "version": "1.0.0",
+  "description": "This is the public repository for <https://circleci.com/docs/>, a static website generated by [Jekyll](https://jekyllrb.com/). If you find any errors in our docs or have suggestions, please follow our [Contributing Guide](CONTRIBUTING.md) to submit an issue or pull request.",
+  "main": "index.js",
+  "scripts": {
+    "webpack-prod": "NODE_ENV=production webpack -p"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/circleci/circleci-docs.git"
+  },
+  "license": "EPL-1.0",
```

Comment:
Is this correct? I just copied this line from somewhere 🙈

Reply:
I believe this is supposed to be [MIT](https://github.com/circleci/circleci-docs/blob/master/LICENSE)